src/InputGroup/InputGroup.js

```diff
@@ -59,6 +59,8 @@ export class InputGroup extends Component {
 
     render() {
         const {
+            inputClassName,
+            inputType,
```

Comment:
I'm getting duplicate declaration errors on this - I think `inputType` is also on line 74

Reply:
oops looks like I merged this wrong. should be fixed with latest push.